src/utils.ts

```diff
@@ -157,3 +157,7 @@ export const hasCjsExtension = (filename: string): boolean =>
 // TODO: add unit test
 export const baseNameWithoutExtension = (filename: string): string =>
   path.basename(filename, path.extname(filename))
+
+export const isTestFile = (filename: string): boolean =>
+  // include .test. or .spec. in filename
+  /\.(test|spec)\./.test(filename)
```

Comment:
need to test `baseNameWithoutExtension`, cause it can be `a.f.foo.spec.some.js` which looks like a random valid case.

```js
/\.(test|spec)$/.test(baseNameWithoutExtension(filename))
```